api/v1alpha08/sonataflowplatform_types.go

```diff
@@ -61,6 +65,15 @@ type SonataFlowPlatformSpec struct {
 	Properties *PropertyPlatformSpec `json:"properties,omitempty"`
 }
 
+// PlatformEventingSpec specifies the broker to be used as the default sink or source
+// +k8s:openapi-gen=true
+type PlatformEventingSpec struct {
+	// Broker to commute with workflow deployment
```

Comment:
```suggestion
	// Broker to commute with workflow deployment. It can be the default broker when the workflow, Dataindex, or Jobservice does not have a sink or source specified.
```